core/src/main/java/nl/nn/adapterframework/jmx/JmxAdapterFilter.java

```diff
@@ -32,16 +32,10 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.jmx.export.MBeanExporter;
 
+import nl.nn.adapterframework.configuration.AdapterProcessor;
 import nl.nn.adapterframework.core.Adapter;
 
-/**
- * This implementation of {@link IAdapterService} registers the adapters to a JMX server.
- * 
- * NOTE: Using the PlatformMBeanServer on WebSphere changes ObjectNames on registration.
- * 
- * @author Niels Meijer
- */
-public class JmxRegisteringAdapterService extends AdapterService implements InitializingBean {
+public class JmxAdapterFilter extends AdapterProcessor implements InitializingBean {
```

Comment:
Ik weet niet welke naam ik beter vind: AdapterFilter of AdapterProcessor. Wel vind ik dat ze niet door elkaar gebruikt moeten worden.

Reply:
AdapterLifecylceWrapper